adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.h

```diff
@@ -23,10 +23,13 @@ NSError *_Nonnull GADMAdapterVungleErrorWithCodeAndDescription(GADMAdapterVungle
                                                                NSString *_Nonnull description);
 
 /// Returns a GADAdSize object that is valid for Vungle SDK.
-GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize);
+GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize, NSString *_Nonnull placementId);
 
-/// Returns a Liftoff Monetize BannerSize from the provided GADAdSize.
-BannerSize GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize);
+/// Returns a Liftoff Monetize VungleAdSize from the provided GADAdSize.
+VungleAdSize *_Nonnull GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize);
```

Comment:
These two functions seem to be no longer used in the adapter. Why not just remove these two?

Reply:
Removed. Thanks